db/migrate/20181108195546_migrate_miq_policy_towhat_to_resource_type.rb

```diff
@@ -0,0 +1,21 @@
+class MigrateMiqPolicyTowhatToResourceType < ActiveRecord::Migration[5.0]
+  class MiqPolicy < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
```

Comment:
it is not needed for `MiqPolicy` as there is no `MiqPolicy#type` column see: https://github.com/ManageIQ/manageiq-schema/issues/4#issuecomment-310643185